libstuff/SSSLState.cpp

```diff
@@ -32,12 +34,39 @@ SSSLState::SSSLState(const string& hostname, int socket) {
         STHROW("mbedtls_ctr_drbg_seed failed with error " + to_string(lastResult) + ": " + errorBuffer);
     }
 
+    // Load OS default CA certificates for peer verification (based on Ubuntu)
+    const string certPath = "/etc/ssl/certs/";
+    lastResult = mbedtls_x509_crt_parse_path(&cacert, certPath.c_str());
```

Comment:
Should probably make this configurable via the environment so we can override with the warp cert if needed